### DISCUSSION TITLE
no such file or directory: powerlevel10k.zsh-theme

### DISCUSSION THREAD

**Discussion by ehackett00** [ohmyzsh/ohmyzsh#9840] [Help] [answered]
I am trying to install Oh My zsh on my Mac.   I am installing in iTerm2  I get the following message in the terminal:

/Users/macbacupfb11/.oh-my-zsh/oh-my-zsh.sh:source:128: no such file or directory: /Users/macbacupfb11/.oh-my-zsh/themes/powerlevel10k.zsh-theme

I have not yet seen the theme.  A few days ago I did see a new image on the screen, but then I was told that Oh My zsh would not install until I fixed some file and folder permissions.  I looked threw the FAQ but did not find a related topic.

Can you help?

Ed

**Comment by mcornella** [comment, marked as answer]
You need to follow the p10k install instructions for Oh My Zsh: https://github.com/romkatv/powerlevel10k#oh-my-zsh.

**Comment by hguitar** [comment]
Disable the current theme in your framework / plugin manager.
- oh-my-zsh: Open ~/.zshrc and remove the line that sets ZSH_THEME. It might look like this: ZSH_THEME="powerlevel9k/powerlevel9k".
- zplug: Open ~/.zshrc and remove the zplug command that refers to your current theme. For example, if you are currently using Powerlevel9k, look for zplug bhilburn/powerlevel9k, use:powerlevel9k.zsh-theme.
- prezto: Open ~/.zpreztorc and put zstyle :prezto:module:prompt theme off in it. Remove any other command that sets theme such as zstyle :prezto:module:prompt theme powerlevel9k.
- antigen: Open ~/.zshrc and remove the line that sets antigen theme. It might look like this: antigen theme powerlevel9k/powerlevel9k.

**Comment by AndresCampuzano** [comment]
I just commented `# source ~/powerlevel10k/powerlevel10k.zsh-theme` at the end of `.zsh` file

**Reply by rahuls** [reply]
Worked for me. Thanks!

**Comment by SH2282000** [comment]
For me, it was the path that magically changed from:
```shell
/opt/homebrew/opt/powerlevel10k/powerlevel10k.zsh-theme
# to
/opt/homebrew/share/powerlevel10k/powerlevel10k.zsh-theme 
```

After changing it in the line that `source` this file, the new terminal windows were again showing the theme.

**Reply by momonga-ml** [reply]
Not sure why this changed 🤔

**Reply by tvb** [reply]
It changed for me after I did a `brew upgrade`

**Reply by memark** [reply]
Same here!

**Reply by bchschaefer** [reply]
I just got this very same problem after a `brew upgrade` a few moments ago. Changing the path as mentioned above, fixed it for me.

**Reply by nraval1729** [reply]
I got the same issue after a `brew upgrade` just now as well. Updating the path resolved it for me as well, thank you!

**Reply by davidysoards** [reply]
thank you!

**Reply by wulfland** [reply]
Same issue. Fixing the path in .zshrc fixed it.

**Reply by ahuggins** [reply]
YUP.. This was it. These things changing without any notice is quite annoying.

**Comment by erose96** [comment]
Updated a bunch of my packages and ended up finding the file in the following directory:
```zsh
/usr/local/opt/powerlevel10k/share/powerlevel10k/powerlevel10k.zsh-theme
```

I just changed the `source` path in my .zshrc to that and it seems to be working

**Comment by codelabs** [comment]
> Updated a bunch of my packages and ended up finding the file in the following directory:
> 
> ```shell
> /usr/local/opt/powerlevel10k/share/powerlevel10k/powerlevel10k.zsh-theme
> ```
> 
> I just changed the `source` path in my .zshrc to that and it seems to be working

This is accurate. The path changed and just updating to its new path, fixed everything

```shell
source $(brew --prefix)/opt/powerlevel10k/share/powerlevel10k/powerlevel10k.zsh-theme
```

**Comment by imranpollob** [comment]
It happened to me after `brew upgrade`. I needed to change the word `opt` to `share` in `.zshrc` file.

source /opt/homebrew/`share`/powerlevel10k/powerlevel10k.zsh-theme
